src/libs/lfs_config.h

```diff
@@ -0,0 +1,234 @@
+#pragma once
+
+#include <libraries/log/nrf_log.h>
+
+#define LFS_TRACE(fmt, ...) NRF_LOG_DEBUG("[LFS] %s:%d:trace: " fmt "\n", __FILE__, __LINE__, ##__VA_ARGS__)
+#define LFS_DEBUG(fmt, ...) NRF_LOG_DEBUG("[LFS] %s:%d:debug: " fmt "\n", __FILE__, __LINE__, ##__VA_ARGS__)
+#define LFS_WARN(fmt, ...) NRF_LOG_WARNING("[LFS] %s:%d:warn: " fmt "\n", __FILE__, __LINE__, ##__VA_ARGS__)
+#define LFS_ERROR(fmt, ...) NRF_LOG_ERROR("[LFS] %s:%d:error: " fmt "\n", __FILE__, __LINE__, ##__VA_ARGS__)
```

Comment:
I'm not sure I like the `##` here. It's a GNU extension, and we generally try to avoid those. We could use the trick used below with the empty string for a good alternative.